openslides_backend/action/user/reset_password_to_default_temporary.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any, Dict
+
+from ...shared.exceptions import ActionException
+from ...shared.patterns import FullQualifiedId
+from ..register import register_action
+from .reset_password_to_default import UserResetPasswordToDefaultAction
+
+
+@register_action("user.reset_password_to_default_temporary")
+class UserResetPasswordToDefaultTemporaryAction(UserResetPasswordToDefaultAction):
+    """
+    Action to reset a password to default of a temporary user.
+    """
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Check for temporary user and call super().update_instance().
+        """
+        user = self.database.get(
+            FullQualifiedId(self.model.collection, instance["id"]), ["meeting_id"]
+        )
+        if not user.get("meeting_id"):
+            raise ActionException(f"User {instance['id']} is not temporary.")
```

Comment:
You should move this check into a base class/mixin since it is needed in every temporary action. Please also adjust existing temporary actions then

Reply:
Created a mixin for the check.